charms/worker/charmcraft.yaml

```diff
@@ -58,6 +72,7 @@ parts:
   charm:
     build-packages: [git]
     charm-entrypoint: k8s/src/charm.py
+    charm-requirements: [k8s/requirements.txt]
```

Comment:
worker charm can specific it's `requirements.txt` so that the python reqs can be installed with `no-binary`